store/tikv/2pc.go

```diff
@@ -764,7 +764,7 @@ func (action actionPessimisticLock) handleSingleBatch(c *twoPhaseCommitter, bo *
 			} else {
 				// the lockWaitTime is set, we should return wait timeout if we are still blocked by a lock
 				if time.Since(lockWaitStartTime).Milliseconds() >= action.LockWaitTime {
-					return ErrLockWaitTimeout
+					return errors.Trace(ErrLockWaitTimeout)
```

Comment:
The `ErrLockWaitTimeout` contains the stack information already.

Reply:
No, it doesn't.
Here it's not `ErrLockWaitTimeout.GenWithStack()`